### ISSUE TITLE
Can not change parent node for plan: IntegrityError: (1048, u"Column 'original_value' cannot be null")

### ISSUE THREAD

**Issue by jhutar** [Nitrate/Nitrate#328]
When I attempt to "Submit" newly selected parent plan, it does nothing in UI, on `./manage.py runserver` console there is:

```
Internal Server Error: /ajax/update/
Traceback (most recent call last):
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/views/decorators/http.py", line 40, in inner
    return func(request, *args, **kwargs)
  File "/home/pok/Checkouts/Nitrate/tcms/core/ajax.py", line 424, in update
    new_value=value)
  File "/home/pok/Checkouts/Nitrate/tcms/core/models/__init__.py", line 51, in log_action
    new_value=new_value)
  File "/home/pok/Checkouts/Nitrate/tcms/core/logs/views.py", line 62, in make
    model.save()
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/base.py", line 924, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/base.py", line 963, in _do_insert
    using=using, raw=raw)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 1112, in execute_sql
    cursor.execute(sql, params)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/pok/Checkouts/Nitrate/.venv/lib/python2.7/site-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
IntegrityError: (1048, u"Column 'original_value' cannot be null")
[12/Sep/2018 08:09:51] "POST /ajax/update/ HTTP/1.1" 500 29586
```

**Comment by tkdchen** [comment]
Same issue #229

**Comment by tkdchen** [comment]
This was already fixed in 60d10c0.
